### PR TITLE
[SSP-24] [UI/Backend] Store Recruitment Page

### PR DESCRIPTION
**Pull request by Fr4nzy**

### CODE REVIEW

File: app/src/main/java/com/projectfkklp/saristorepos/activities/store_profile/StoreProfilePage.java
Comment (mzowera) [inline app/src/main/java/com/projectfkklp/saristorepos/activities/store_profile/StoreProfilePage.java:1]: Remove unused imports

File: app/src/main/java/com/projectfkklp/saristorepos/managers/UserStoreRelationManager.java [hunk 1]
```diff
@@ -45,6 +46,31 @@ public static void request(Context context, Store store, UserRole userRole, OnSe
...
 
+    public static void invite(Context context, User user, UserRole userRole, OnSetFirebaseDocument onSetFirebaseDocument){
+        String id = ModelUtils.createUUID();
+        UserStoreRelation userStoreRelation = new UserStoreRelation(
+                id,
+                SessionRepository.getCurrentUser(context).getId(),
+                user.getId(),
+                userRole,
+                UserStatus.REQUESTED
+        );
```
Comment (mzowera) [inline app/src/main/java/com/projectfkklp/saristorepos/managers/UserStoreRelationManager.java]: Check the detail from store recruitment perspective, change the code

```java
        UserStoreRelation userStoreRelation = new UserStoreRelation(
                id,
                user.getid(),
                SessionRepository.getCurrentStore(context).getId(),
                userRole,
                UserStatus.INVITED
        );
```

**Review by mzowera — CHANGES_REQUESTED** [review]

**Review by mzowera — CHANGES_REQUESTED** [review]
1. Searching by name is not working, please fix (double check search by gmail and phone number)
2. Associated Users should not be included in search results

File: app/src/main/java/com/projectfkklp/saristorepos/managers/UserStoreRelationManager.java (hunk 1 above)
Comment (Fr4nzy) [reply app/src/main/java/com/projectfkklp/saristorepos/managers/UserStoreRelationManager.java]: image

**Comment by Fr4nzy** [comment]
> 1. Searching by name is not working, please fix (double check search by gmail and phone number)
> 2. Associated Users should not be included in search results

image

**Review by mzowera — APPROVED** [review]

**Pull request merged**